corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -174,7 +174,7 @@
               <span class="hq-help-template"
                     data-content='{% blocktrans %}
                                     <p>Checking this will add a button to download a configuration file
-                                       for the <a href="https://confluence.dimagi.com/display/commcarepublic/CommCare+Data+Export+Tool" target="_blank">
+                                       for the <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143955952" target="_blank">
```

Comment:
nit: Seems like we're including friendly names in links, in which case this would be:
```suggestion
                                       for the <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143955952/CommCare+Data+Export+Tool+DET" target="_blank">
```